models2/CommentModel.es6.js

```diff
@@ -27,6 +27,7 @@ export default class CommentModel extends RedditModel {
     replies: T.array,
     numReplies: T.number,
     loadMore: T.bool,
+    loadMoreIds: T.array,
```

Comment:
put it on the [jira board](https://reddit.atlassian.net/secure/RapidBoard.jspa?rapidView=105&view=planning.nodetail&epics=visible&selectedEpic=MOBILEWEB-575)!